### PR TITLE
OPSEXP-1956: add configuration for mTLS auth between repo and tengine (community setup)

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1956

Leverage roles' entrypoints in java role to specifically deploy keystores rather than making it a conditional task inclusion (offers better argument validation)
Also fixes a missing dependency in transformers (exiftools)

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ nexus_repository:
 acs:
   artifact_name: alfresco-content-services-distribution
   edition: Enterprise
-  repository: "{{ nexus_repository.enterprise_releases }}"
-  version: 7.4.0-M3
+  repository: "{{ nexus_repository.development_releases }}"
```
Comment (gionn) [inline group_vars/all.yml:22]: still need to find a solution for this - and avoid merging this on master

for now maybe add a # FIXME

File: molecule/pki/host_vars/tls-ats-instance.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# placeholder
+ansible_user: ansible
+known_urls:
+  - http://localhost:8080/share
+  - https://someotherapp/
+acs:
+  artifact_name: alfresco-content-services-community-distribution
+  edition: Community
+  repository: "{{ nexus_repository.development_releases }}"
+  version: 7.4.0-A27
```
Comment (gionn) [inline molecule/pki/host_vars/tls-ats-instance.yaml:10]: we should avoid having versions in molecule tests or we will continue to forget about them - why is necessary here?

File: roles/transformers/tasks/main.yml [hunk 3]
```diff
@@ -8,6 +8,31 @@
...
+  yum_repository:
+    name: epel
+    description: EPEL YUM repo
+    metalink: >-
+      {{ epel_meta_base }}?repo=epel-{{ epel_ver }}{{ epel_meta_args }}
+    gpgcheck: true
+    gpgkey: >-
+      {{ epel_gpg_base }}/RPM-GPG-KEY-EPEL-{{ epel_ver }}
+  when: pkg_format == 'rpm'
+
```
Comment (gionn) [inline roles/transformers/tasks/main.yml:35]: move to common role?

**Review by gionn — CHANGES_REQUESTED** [review]
I see cherry-picked commits from other PR, that means this PR should get merged AFTER them

File: molecule/pki/host_vars/tls-ats-instance.yaml (hunk 2 above)
Comment (alxgomz) [reply molecule/pki/host_vars/tls-ats-instance.yaml:10]: leftovers. I had this file previously during development as I thought I would make this scenario a community one. I gave up with this idea. Will remove.

**Review by gionn — APPROVED** [review]

**Pull request merged**